mlrun/api/launcher.py

```diff
@@ -35,12 +35,13 @@
 class ServerSideLauncher(launcher.BaseLauncher):
     def __init__(
         self,
+        is_remote: bool,
         local: bool = False,
```

Comment:
sounds like they mean the same
not remote === local

Reply:
we have this issue in multiple places, for now I will clarify with a comment

Reply:
decided to remove this validation because for some reason nuclio runtime is not remote (even though it is called RemoteRuntime...) and it messes up everything